schema_ideas.md

```diff
@@ -0,0 +1,29 @@
+## 2 Tables:
+
+  
+
+## Users Table:
+
```

Comment:
This is a really good idea. Helps to get your schema down on paper or in a markdown file before you start coding that schema 👍